layouts/_partials/recent-articles.html

```diff
@@ -1,8 +0,0 @@
-{{ if .Site.Params.homepage.showRecent | default false }}
```

Comment:
We don't use this file right now, and with `recent-posts` instead, I thought better to remove this to not be confusing